thunder/tests/test_nvfuser.py

```diff
@@ -991,3 +991,78 @@ def make_integer_tensor():
         rout = f(x.cpu(), y.cpu()).to(device)
         jout = f(x, y)
         assert rout.equal(jout)
+
+
+@instantiate(
+    dtypes=(thunder.float16, thunder.bfloat16),
+    devicetypes=(devices.DeviceType.CUDA,),
+    executors=(nvFuserExecutor,),
+    decorators=(
+        pytest.mark.skipif(
```

Comment:
I wonder if the `ops` decorator is more appropriate here, so we can reuse the existing `sdpa` opinfo.

Example with cudnn executor. https://github.com/Lightning-AI/lightning-thunder/blob/main/thunder/tests/framework.py#L362

Reply:
I considered that. I was concerned about there being no case where Flash Attention is the chosen backend (since they had different E, Ev) and the test silently not checking anything.
So created the test in the current format.